src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,14 @@
+<hibernate-configuration>
+    <session-factory>
+        <property name="connection.url">jdbc:mysql://localhost:3306/book?serverTimezone=UTC</property>
+        <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+        <property name="connection.username">root</property>
+        <property name="connection.password">Keksmaestro2012</property>
```

Comment:
Never commit your real credentials, always empty these fields or place some dummy text instead of real credentials